### DISCUSSION TITLE
How about outputting to a bluetooth speaker

### DISCUSSION THREAD

**Discussion by clendee** [GeekFunkLabs/fluidpatcher#30] [Ideas]
Since the Pi has built-in bluetooth, how difficult woud it be to implement the option of scanning for bluetooth devices on the Squishbox, ie bluetooth headphones and speakers.

I realise that its very easy to come up with ideas, and implemening them can be quite a chore.  As I'm not a developer, merely and enthusiast, I appreaciate all the work that has gone into this device and think its amazing.

I had a quick look into connecting bluetooth devices from the command line and this covered a few situations, at least enough to whet my appetite.
https://computingforgeeks.com/connect-to-bluetooth-device-from-linux-terminal/

**Comment by albedozero** [comment]
Sounds like an interesting trick to try out, and should be possible - maybe a weekend project/video topic. Not to be too much of a pessimist, but I'm guessing the audio latency will be too hefty for playing live.

**Reply by clendee** [reply]
Yeah, its a bit of a tough call like that alright I guess.  Its different to just casting a song to bluetooth where the latency doesnt matter.
